workspaces/server/src/routes/calendar/events.ts

```diff
@@ -92,17 +93,36 @@ const route = app
 				locationId,
 			};
 			try {
+				// もしAdminじゃないなら、自分のイベントだけ更新できる
+				const roleIds = c.get("roleIds");
+				if (!roleIds.includes(ROLE_IDS.ADMIN)) {
+					const event = await CalendarRepository.getEventById(id);
+					if (event.userId !== userId) {
+						return c.json({ error: "not authorized" }, 403);
```

Comment:
```suggestion
						return c.json({ error: "Forbidden" }, 403);
```
not authorized = unauthorized は 401 じゃない？